x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/utils/LoggingDeprecationAccumulationHandler.java

```diff
@@ -29,15 +29,15 @@
     public void usedDeprecatedName(String usedName, String modernName) {
         LoggingDeprecationHandler.INSTANCE.usedDeprecatedName(usedName, modernName);
         deprecations.add(LoggerMessageFormat.format("Deprecated field [{}] used, expected [{}] instead",
-            usedName,
+            (Object)usedName,
```

Comment:
This confused me until I realised it was the var args argument. Is `new Object[] {usedName, modernName}` any clearer or confusing in a different way

Reply:
Yeah, these two conflicting methods are frustratingly designed. I could probably make it clearer with your suggestion.